orunmila.go

```diff
@@ -276,12 +276,26 @@ func describeSubcmd(args []string) {
 	path, err := os.Getwd()
 	check(err)
 
-	flag := flag.NewFlagSet("describe", flag.ContinueOnError)
+	flag := flag.NewFlagSet("describe", flag.ExitOnError)
+
+	flag.Usage = func() {
+		fmt.Fprint(flag.Output(), "Describe the database\n\n")
+		fmt.Fprintf(flag.Output(), "Usage of orunmila descibe:\n")
+		flag.PrintDefaults()
+		fmt.Fprintln(flag.Output(), "  words strings\n\tspace separated list of words to add")
+		fmt.Fprintln(flag.Output(), "\nexample: orunmila describe My Awesome Description")
+	}
+
 	var (
 		dbPtr = flag.String("db", filepath.Join(path, "orunmila.db"), "the database filename (default: orunmila.db)")
 	)
 	flag.Parse(args)
 
+	if len(args) < 1 {
+		log.Errorln("please provide a description")
+		flag.Usage()
+		os.Exit(1)
```

Comment:
as of the `os.Exit()` calls it seems that the `flag` package handles it already when `flag.ExitOnError` is set in the `flag.NewFlagSet()`

for reference: https://github.com/campoy/embedmd/pull/12
https://pkg.go.dev/flag#pkg-variables (3rd example)

Reply:
nop this is needed. The reference is because they used os.Exit(1) from Inside the Usage() function.
https://github.com/campoy/embedmd/pull/12/files